components/Registration/index.tsx

```diff
@@ -226,43 +226,30 @@ const Registration: FunctionComponent<RegistrationProps> = ({
       stakingAddress: string,
       tokenAddress: string,
       userWalletAddress: string,
-      contractAddress: string
+      contractAddress: string,
     ): Promise<TransactionResponse> => {
-      try {
-        console.log('calling autostake method');
-
-        console.log('stakingAddress', stakingAddress);
-        console.log('tokenAddress', tokenAddress);
-        console.log('contractAddress', contractAddress);
-
-        const txn = await airdropContract.stake(
-          contractAddress,
-          tokenAddress,
-          stakingAddress,
-          stakeDetails.airdrop_rewards.toString(),
-          stakeDetails.stakable_tokens.toString(),
-          activeWindow.airdrop_id?.toString(),
-          activeWindow.airdrop_window_id?.toString(),
-          signature
-        );
-        return txn;
-      } catch (error: any) {
-        console.log('contract error', error);
-
-        throw error;
```

Comment:
Why have we removed the exception handing part ?

Reply:
Cause its not doing anything. You are catching an error to throw it again